.changeset/fluffy-bananas-remember.md

```diff
@@ -0,0 +1,5 @@
+---
+"@patternfly/pfe-core": major
```

Comment:
```suggestion
"@patternfly/pfe-core": patch
```

Reply:
Oops didn't mean to select major there, good catch 🙃